test/unit/payments/payment-flow-builder.spec.ts

```diff
@@ -70,14 +71,21 @@ describe("LightningPaymentFlowBuilder", () => {
   const immediateSpread = 0.001 // 0.10 %
   // const futureSpread = 0.0012 // 0.12%
 
+  const centsFromSatsForMid = ({ sats, spread, round }): bigint => {
```

Comment:
This behavior is indirectly enforced in main code here: https://github.com/GaloyMoney/galoy/blob/6eb60ae54f6095d5791941e7492f1177c7d6746a/test/unit/domain/payments/price-ratio.spec.ts#L242